code-challenges.js

```diff
@@ -39,13 +64,34 @@ const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const combineArrays = (Array1,Array2) =>{
+    /* The purpose of this function is to combine two arrays and return the of COMBINED array.
+    Eg: const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+        const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+        Expected output: 9
+    */ 
+    res = Array1.concat(Array2)
+    return res.length
+}
```

Comment:
Make sure your pseudo is explaining the functionality of your code! The purpose is great, but explaining what the code is doing and how it's doing it is the purpose of pseudo code. Also, remember that in JavaScript, we use camelCase for parameters. Lastly, remember that if you are declaring a variable, use a variable declaration keyword. Otherwise, love that you made a dynamic function, and love that you made 2 parameters to allow for 2 arguments, well done!